packages/design-system/src/components/tooltip/index.js

```diff
@@ -128,23 +128,58 @@ function Tooltip({
     }),
     [placement]
   );
+  // We can sometimes render a tooltip too far to the left, ie. in RTL mode.
+  // when that is the case, we can switch to placement-start and the tooltip will no longer get cutoff.
+  const updatePlacement = useCallback(() => {
+    const currentPlacement = placementRef.current;
+    switch (currentPlacement) {
+      case PLACEMENT.BOTTOM_START:
+      case PLACEMENT.TOP_START:
+      case PLACEMENT.RIGHT_START:
+        // {placement}-START shouldn't ever appear in overflow so do nothing
+        break;
```

Comment:
good to be explicit. Leaving it as the default without a comment may be confusing.

Reply:
Yeh, I felt like if I didn't leave a comment others would be confused why they weren't accounted for.